tests/test_diffusion.py

```diff
@@ -123,7 +123,7 @@ def execute_devito(ui, dx=0.01, dy=0.01, a=0.5, timesteps=500):
     dt = dx2 * dy2 / (2 * a * (dx2 + dy2))
     # Allocate the grid and set initial condition
     # Note: This should be made simpler through the use of defaults
-    u = TimeData(name='u', shape=(nx, ny), time_order=1)
+    u = TimeData(name='u', shape=(nx, ny), time_order=1, space_order=2)
```

Comment:
Having the order as a property of the data object makes me a uncomfortable since the order is a logical component of the operator. Considering its normal to expect data objects to work across multiple operators, this seems like a confusing interface choice.